hook.go

```diff
@@ -43,6 +43,25 @@ func NewHook(fileName string) (*Hook, error) {
 	}, nil
 }
 
+// CheckHook checks the given map against HookMapABI to have appropriate values
+// Only checks Type, KeySize, ValueSize
+func CheckHook(m *ebpf.Map) error {
```

Comment:
NIT: We refer to the maps as `hookMap` everywhere else, can we rename this function to `CheckHookMap` for consistency?

Reply:
Does the function even need to be exported?

Reply:
Unfortunately `cmd/xdpcap` uses it too. I guess we could move it to an `internal/` package.

Reply:
I wasn't sure which parts of `hook.go` should be moved to `internal/`package so I didn't move it.

Reply:
Just `CheckMapHook()` I think, the ABI was previously exported so we can leave it as is :)

Reply:
moved